crime-commons-mod-schemas/src/main/resources/schemas/meansassessment/apiUpdateMeansAssessmentRequest.json

```diff
@@ -25,11 +25,18 @@
     "fullAssessmentNotes": {
       "type": "string",
       "description": "Full assessment notes field."
+    },
+    "incomeEvidence": {
+      "type": "array",
+      "description": "Income evidence.",
+      "items": {
+        "$ref": "common/apiIncomeEvidence.json"
+      }
     }
   },
   "extends": {
     "$ref": "apiMeansAssessmentRequest.json"
   },
   "additionalProperties": false,
-  "required": ["initTotalAggregatedIncome"]
+  "required": ["fullAssessmentDate", "initTotalAggregatedIncome"]
```

Comment:
apiUpdateMeansAssessmentRequest is used to update means assessment, when updating an Initial assessment do we have fullAssessmentDate?

Reply:
No, I have removed this now - I deleted a duplicate schema and this was carried across by accident.